app/test/unit/git/merge-test.ts

```diff
@@ -82,7 +81,29 @@ describe('git/merge', () => {
         'master',
         'origin/some-unknown-branch'
       )
-      expect(ref).is.null
+      expect(ref).toBeNull()
+    })
+  })
+  describe('abortMerge', () => {
+    let repository: Repository
+    const subject = () => abortMerge(repository)
+    describe('when there is no in-progress merge', () => {
+      beforeEach(async () => {
+        repository = await setupEmptyRepository()
+      })
+      it('throws an error', async () => {
+        await expect(subject()).rejects.toThrow(
+          /There is no merge in progress, so there is nothing to abort/
+        )
+      })
+    })
+    describe('in the middle of resolving conflicts merge', () => {
+      beforeEach(async () => {
+        repository = await setupConflictedRepo()
+      })
+      it('aborts the merge', async () => {
+        await expect(subject()).resolves.not.toThrow()
```

Comment:
A suggestion for the next time you're in here - what about also verifying that the Git repository is no longer in a conflicted state?